Add bottom to top button

mrvishalsharma01 opened the pull request:
i hope you liked my work can i get some more task?

DharshiBalasubramaniyam commented:
@mrvishalsharma01, Please check the preview once again. I cannot see the bottom to top button

mrvishalsharma01 commented:
I am sorry i don't know what's strong with this i checked many times and it's working perfectly fine....Please guide me if i am wrong.

**The steps i followed:-**
1) I forked your respository
2) Cloned in my system
3) Made a new branch dev 
4) Added images html Css and javaScript code 
5) then i check the git status 
6) then i added all file with proper commit message
7) then again i checked the git status
8) then i pushed the code 

Can you please tell me did i miss something?

**Note:-** Bottom to top icon will only show when we will go to the bottom of the page to come to the top 

Please guide me if i miss something 



https://github.com/user-attachments/assets/b2995619-743f-4c2e-9ee2-8439e089b4f5

DharshiBalasubramaniyam commented:
@mrvishalsharma01, No worries. I think there was a problem in the preview display. The button is correcting working. Thank you for your work!
